deploy_ccf.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the Apache 2.0 License.
+
+set -e
+
+if [ -z "$1" ]; then
+    echo "The application enclave file should be specified (e.g. libloggingenc)"
```

Comment:
I think its more user friendly to allow (/expect) the full path to the enclave here, so that autocomplete works. I think the only reason that we take only the prefix at the Python level is to support virtual enclaves. We should be able to tweak `path.py` so it can also accept the full path (ie - `build_lib_path` can take `libloggingenc` or `libloggingenc.so.signed` and will return `libloggingenc.so.signed`, unless the file is fully specified and doesn't match `enclave_type`, in which case it raises an error).